src/utils/addresses.ts

```diff
@@ -95,9 +95,14 @@ export const getAddressAvailableBalance = (address: Address): bigint =>
 export const selectAddressTransactions = (
```

Comment:
NIT: Not part of the PR I know, but this follows the selector naming scheme, so I'd expect this to be a standalone redux selector. I know it's used in multiple slices, but I find it weird that it's not located in the "store" folder. WDYT?

Reply:
Good point. I think our file structure needs to be adapted to match the new one in the desktop wallet (which is still not set in stone actually). I'll add a comment about this in the next PR.